cypress/integration/adminFlows/pages/landingPage.spec.js

```diff
@@ -2,9 +2,27 @@ describe('Set a landing page from the admin portal', () => {
   beforeEach(() => {
     cy.testSetup();
     cy.fixture('users/adminUser.json').as('user');
-
     cy.get('@user').then((user) => {
       cy.loginUser(user).then(() => {
+        // Ensure Forem instance is private
```

Comment:
Having this in the `beforeEach` closure isn't my favorite approach but somehow I couldn't get it to work with `before`.

Reply:
When you had this in the `before` hook were you also still calling `testSetup` in the `beforeEach`? If so that would reset the changes you made in `before`, and possibly the root of the issue.

Given this is just setup, would it be possible to get the same functionality by sending a POST request from the beforeEach hook? We have similar commands set up in Cypress like `createArticle` (in `cypress/support/commands.js`) to help us set up individual tests. Just thinking it would be a little bit faster especially if we have to run it for multiple tests.

Reply:
> Given this is just setup, would it be possible to get the same functionality by sending a POST request from the beforeEach hook? 

Unfortunately not. To quote from the above comment:

> [S]ince we removed the config API during the `SiteConfig` split there is now no easy way to mark a Forem as private. 

I can give `before` another shot tomorrow.

Reply:
Ah sorry I didn't parse that comment about the config API fully before 👍 

I'm happy if we want to keep this in `beforeEach` to simplify things. Having it in `before` may cause some headaches if the rest of the state needs reset in `testSetup` 🤷‍♀️

Reply:
Thanks, @aitchiss, I'll merge it as-is for now then, we can always reset this spec at a later time. The `beforeEach` variant definitely was the least flaky and most robust in my local tests.